assets/src/dashboard/app/index.js

```diff
@@ -65,19 +65,36 @@ const AppContent = () => {
       currentPath,
       queryParams: { id: templateId },
     },
+    actions: { push },
   } = useRouteHistory();
 
-  const { currentTemplate } = useApi(
+  const { currentTemplate, addInitialFetchListener } = useApi(
     ({
+      actions: {
+        storyApi: { addInitialFetchListener },
+      },
       state: {
         templates: { templates },
       },
     }) => ({
       currentTemplate:
         templateId !== undefined ? templates[templateId]?.title : undefined,
+      addInitialFetchListener,
     })
   );
 
+  // Direct user to templates on first load if they
+  // have no stories created.
+  useEffect(
+    () =>
+      addInitialFetchListener?.((storyStatuses) => {
+        if (storyStatuses?.all <= 0) {
+          push(APP_ROUTES.TEMPLATES_GALLERY);
```

Comment:
To echo Pascal, on line 129 here (`trackScreenView(dynamicPageTitle);` ) this is getting sent to telemetry to track views, we'd want to block the initial 'my stories' tracking and just submit the explore templates tracking to avoid redundancy.

Reply:
okee, should be fixed now